sources/platform/actors/publishing/index.mdx

```diff
@@ -5,39 +5,88 @@ sidebar_position: 7.5
+- Direct exposure to organic user traffic
+- Flexible sharing options with public or private source code settings
+- Complete API documentation and integration examples
+
+<details>
+    <summary>Publishing on Apify significantly streamlines the traditional SaaS development process</summary>
+    | Traditional SaaS Requirements | Handled by Apify Platform |
+    |------------------------------|---------------------------|
+    | Product development | ✓ Actor development only |
+    | Extensive documentation | ✓ README file |
+    | Domain purchase & setup | ✓ Automatic landing page |
+    | Cloud infrastructure | ✓ Built-in scaling |
+    | Payment processing | ✓ Multiple payment options |
+    | Marketing campaigns | ✓ Store marketplace presence |
+    | Sales operations | ✓ Automated transactions |
+</details>
+
+### Monetization options
+
+The platform supports various pricing models to match your Actor's value proposition:
 
-As the name implies, Public Actors are available to the public on [Apify Store](https://apify.com/store), which means that an unmaintained public Actor could negatively affect all the other users that depend on it for their own activities on the platform.
+- Fixed rental fee for continuous access
+- Pay-per-result for usage-based pricing
+- Event-based pricing for specific operations
+- Custom pricing strategies for enterprise clients
 
-Public Actors are regularly submitted to [automated tests](/platform/actors/publishing/test) to ensure they are functioning properly. Before making an Actor public, we recommend you reserve enough time to maintain the project (~2 hours weekly). This will ensure that your Actor maintains its long-term quality, improving your chances of successfully [monetizing your Actors](./monetize.mdx).
+## Maintain public Actors
+
+While private Actors don't necessarily require ongoin maintenance, public Actors demand a highres level of resonsibility. Since public Actors are available on the [Apify Store](https://apify.com/store) and may be used by other users, an unmaintained public Actor could negatively impact those who depend on it.
+
+To ensure long-term quality and improve your chances of successfully monetizing your Actors, we rocemmend reserving approximately 2 hours per week for maintenance tasks, such as:
+
+- Addressing bug reports & issues
+- Implementing updates & improvements
+- Responding to user inquireies & feedback
 
  <img src={require("./images/apify-store.png").default} title="Apify Store" width="60%" />
 
-If you decide to make your Actor code publicly available on [GitHub](https://github.com), its quality becomes a priority since your Actor could be a new user's first experience with Apify.
+If you decide to make your Actor's code publicly available on [GitHub](https://github.com), code quality becomes even more crucial, as your Actor may be the first experience some users have with Apify.
+
+### Handling breaking changes
```

Comment:
Here we should provide some best practices regarding versioning the Actors, e.g. use `version-X` tags. Do we have some page about that?

Reply:
Closest page where we describe versioning is [here](https://docs.apify.com/platform/actors/development/builds-and-runs/builds). But I guess that is more of an explanation than any kind of best practices, not sure if that would be suitable